src/handlers.js

```diff
@@ -389,25 +418,8 @@ async function jobHandler(message) {
     taskGroupId = graphConfig.tasks[0].task.taskGroupId;
     debug(`Creating tasks for ${organization}/${repository}@${sha} (taskGroupId: ${taskGroupId})`);
     await this.createTasks({scopes: graphConfig.scopes, tasks: graphConfig.tasks});
-  } catch (e) {
-    debug(`Creating tasks for ${organization}/${repository}@${sha} failed! Leaving comment on Github.`);
-    groupState = 'failure';
-    await this.createExceptionComment({instGithub, organization, repository, sha, error: e});
-  } finally {
-    debug(`Trying to create status for ${organization}/${repository}@${sha} (${groupState})`);
-    let eventType = message.payload.details['event.type'];
-    let statusContext = `${context.cfg.app.statusContext} (${eventType.split('.')[0]})`;
-    let description = groupState === 'pending' ? `TaskGroup: Pending (for ${eventType})` : 'TaskGroup: Exception';
-    const target_url = libUrls.ui(context.cfg.taskcluster.rootUrl, `/task-group-inspector/#/${taskGroupId}`);
-    await instGithub.repos.createStatus({
-      owner: organization,
-      repo: repository,
-      sha,
-      state: groupState,
-      target_url,
-      description,
-      context: statusContext,
-    });
+
+    debug(`Trying to create status for ${organization}/${repository}@${sha} (${groupState}) in Azure`);
```

Comment:
Is this still accurate?

Reply:
I stuck "in Azure" at the back so it would be accurate. I'll change the wording so that it's clearer

Reply:
Sorry, I should have been more specific: instead of creating anything, it now sends a pulse message..

Reply:
Well, it creates a record in the db, that's what I wanted this log message to say

Reply:
Oh, I get it now :)  How about "Add .. to Builds table"?